### ISSUE TITLE
[Feature Request] Loomio

### ISSUE THREAD

**Issue by marxistvegan** [syncloud/platform#452]
Hey folks, 
Not sure if there are folks thinking about this or maybe nextcloud suffices to fulfill this need, but as an organizer I find that something akin to project management would be a good tool and I think Loomio would be a really good way to achieve this. Thoughts?

Documentation on deploying it can be found here https://github.com/loomio/loomio-deploy

**Comment by cyberb** [comment]
Sounds good, have you used it yourself?

**Comment by marxistvegan** [comment]
> Sounds good, have you used it yourself?

Yeah I have and still use it too... it is a good software platform, worth doing their trial run to see it in action.

**Comment by cyberb** [comment]
Now I remember diaspora moved out of it to discourse once.
https://www.loomio.org/d/Agop3EXq/outgrowing-loomio

Discourse: https://github.com/discourse/discourse seems to be much more popular.

**Comment by marxistvegan** [comment]
Let me see, I will test out discourse demo.
